tests/omhttp-batch-jsonarray-fail-often.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# This file is part of the rsyslog project, released under ASL 2.0
+port=8080
```

Comment:
We need to decide if these tests here shall be able to run in parallel or not. Depends a bit on ressource usage. If running in parallel, we need dynamic port (there is plumbing for this), if not, we need to serialize via Makefile (like elasticsearch).